client/src/components/Sub_Questions/QuestionEntry.jsx

```diff
@@ -0,0 +1,60 @@
+import React, { useState, useEffect } from 'react';
+import { parseISO, format } from 'date-fns';
+import axios from 'axios';
+import AnswerEntry from './AnswerEntry.jsx';
+
+export default function QuestionEntry({ question }) {
+  const [allAnswers, setAllAnswers] = useState([]);
+  const [currentAnswers, setCurrentAnswers] = useState([]);
+  const [clicked, setClicked] = useState(false);
+  const [moreAnswers, setMoreAnswers] = useState(false);
+
+  useEffect(() => {
+    axios.get(`/api/qa/questions/${question.question_id}/answers?page=1&count=5`)
+      .then(({ data: { results } }) => {
+        setAllAnswers(results);
+      })
+      .catch((err) => {
+        console.log(err);
+      });
+  }, []);
```

Comment:
same question, may need to pass the question as the second argument of useEffect so the component will re-render every time the question changes.